source/Calamari.Aws/Util/ConstantClassExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Amazon.Runtime;
+
+namespace Calamari.Aws.Util
+{
+    public static class ConstantHelpers
+    {
+        public static IEnumerable<T> GetConstantValues<T>() where T : ConstantClass
```

Comment:
The AWS SDK provides these constant value objects as static members of a class, but provide no way of getting a definitive list of these.